src/commands/upload.js

```diff
@@ -51,8 +53,9 @@ module.exports = {
 
       return promise
         .then(result => {
+          const artifactUrl = zeus.transport.getUrl(result.download_url);
```

Comment:
This is accessing a private member. We need to add `getUrl` as public method to `Client` or, better, make the URL returned by the endpoint in Zeus absolute. I'd prefer the last one, actually.